extension/json/src/reader/buffered_json_reader.cpp

```diff
@@ -40,6 +40,21 @@ BufferedJsonReader::BufferedJsonReader(main::ClientContext& context, std::string
     fileHandle = std::make_unique<JsonFileHandle>(std::move(fileInfo));
 }
 
+std::string BufferedJsonReader::reconstructLine(uint64_t startPosition, uint64_t endPosition) {
+    KU_ASSERT(endPosition >= startPosition);
+
+    std::lock_guard<std::mutex> lockGuard{lock};
+
+    std::string ret;
+    ret.resize(endPosition - startPosition);
+
+    bool finishedFile = false;
+    fileHandle->readAtPosition(reinterpret_cast<uint8_t*>(ret.data()), ret.size(), startPosition,
+        finishedFile);
+
+    return StringUtils::ltrimNewlines(StringUtils::rtrimNewlines(ret));
```

Comment:
Our test runner and shell both don't deal well outputting with multiline values (which JSON records can be). Does anyone have suggestions on how to handle this? For now I'm thinking of just removing newlines from the `skipped_line_or_record` entries